src/app/tests/integration/chip_im_initiator.cpp

```diff
@@ -264,8 +270,6 @@ void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aC
 
     printf("Command Response: %" PRIu64 "/%" PRIu64 "(%.2f%%) time=%.3fms\n", gCommandRespCount, gCommandCount,
```

Comment:
Please move this printf to the new callback on response processed

Reply:
Please also update the PR comments to reflect this change, thanks

Reply:
That is my initial plan, but after I take another look. I found DispatchSingleClusterCommand is only called when CHIP_END_OF_TLV is received, and CommandResponseProcessed is called as long as the response message is processed, it does not check if the message is the end of TLV or not.

Should we still keep the following logic in DispatchSingleClusterCommand? so that the response is not counted if the message is not the end of TLV.

    gCommandRespCount++;

    printf("Command Response: %" PRIu64 "/%" PRIu64 "(%.2f%%) time=%.3fms\n", gCommandRespCount, gCommandCount,
           static_cast<double>(gCommandRespCount) * 100 / gCommandCount, static_cast<double>(transitTime) / 1000);
}